examples/with-docker/package.json

```diff
@@ -2,17 +2,14 @@
   "name": "with-docker",
   "version": "1.0.0",
   "scripts": {
-    "dev": "next",
+    "dev": "next dev",
     "build": "next build",
-    "start": "next start",
-    "build-docker": "docker build -t next-app .",
-    "build-docker-multistage": "docker build --target production -t next-app -f ./Dockerfile.multistage .",
-    "run-docker": "docker run --rm -it -p 3000:3000 next-app"
+    "start": "next start -p $PORT"
```

Comment:
This "start" fails if you're following the "Using Docker" instructions.

Reply:
Thanks for the info! Would you mind making a PR with the changes needed? 🙏